trl/trainer/ppo_config.py

```diff
@@ -53,6 +53,9 @@ class PPOConfig(OnPolicyConfig):
             Discount factor.
```

Comment:
This modif shouldn't be here, right?

Reply:
oh my bad i'll fix it right now

Reply:
You still have modification in ppo files